Changelog.md

```diff
@@ -11,6 +11,7 @@
 - Add button to assignment's annotations tab to allow instructor to download one time annotations (#5088)
 - Removed AssignmentStats table (#5089)
 - Display assignment totals on the grade summary table rounded up to 2 decimal places (#5123)
+-Added a delete button to notes under an assignment submission and removed user_can_modify under note model (#5128)
```

Comment:
Use a space after the `-`. Rephrase "to notes" as "to notes dialog" and "an assignment submission" to "results edit view". Refer to instance methods as `Class#method`, e.g. `Notes#user_can_modify`.